utt/privacy-wallet-service/proto/api/v1/wallet-api.proto

```diff
@@ -101,6 +101,23 @@ message GetStateResponse {
     map<string, uint64> coins = 4;
 }
 
+message SetAppDataRequest {
+    string key = 1;
```

Comment:
Idea of having flexible or transparent KV is good. Few comments:
 - It is better to make this repeated KV pair (or) map. Since single key update would not be flexible.
 - value could be blob but it really depends on what we want to store so not keen on this at this point of time.

Reply:
we could also just force this to be single key/value pair where value is serialized app data (blob) which is specific to application.

Reply:
extedned it to support multiple KV in single message